charts/jenkins/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version 1.5.7 is auto generated based on git commits. Those include a reference to the git commit to be able to get more details.
 
+## 2.6.0 First release in jenkinsci GitHub org
```

Comment:
Should this file get migrated to the repository root? if not, it might make sense to reference it from README

Reply:
It\s now referenced from the chart README.